functions/go/set-namespace/transformer/namespace.go

```diff
@@ -25,8 +25,8 @@ func SetNamespace(rl *fn.ResourceList) (bool, error) {
 	// Get "namespace" arguments from FunctionConfig
 	err := tc.Config(rl.FunctionConfig)
 	if err != nil {
-		rl.Results = append(rl.Results, fn.ErrorConfigObjectResult(err, rl.FunctionConfig))
-		return true, nil
+		rl.Results = append(rl.Results, fn.ErrorResult(err))
+		return false, nil
```

Comment:
We should document the `ResourceListProcessor` interface in the SDK repo. Otherwise, users may not know what does the bool mean here.
https://pkg.go.dev/github.com/GoogleContainerTools/kpt-functions-sdk/go/fn#ResourceListProcessor

Reply:
SG! I'm adding examples and tests on the sdk. (though that PR keeps on postponed by May blocker issues).